### DISCUSSION TITLE
Is there a good solution for seeding the database in wp-env

### DISCUSSION THREAD

**Discussion by johnhooks** [WordPress/gutenberg#49522] [Q&A] [answered]
I want to seed the database of the development environment in `wp-env` using `fakerphp/faker`. I was curious how best to do this. I don't really want the logic to be included in the published plugin. I would prefer it to just be a script. But then I don't have access to the `$wpdb` variable when outside of WordPress. Do I have to just make a standard database query or is there someway require `$wpdb` from outside of WordPress?

**Comment by margolisj** [comment, marked as answer]
You could try using wp cli: https://developer.wordpress.org/cli/commands/db/

I believe it is included with `wp-env` docker.

**Reply by johnhooks** [reply]
This is exactly the answer I found, thanks for posting it here though, so others can find it too.
